spellbook/models/tokens/ethereum/tokens_ethereum_schema.yml

```diff
@@ -54,7 +54,7 @@ models:
       - name: standard
         tests:
           - accepted_values:
-              values: ['erc721', 'erc1155' , 'erc20', 'cryptopunks','superrare']
+              values: ['erc721', 'erc1155' , 'erc20', 'cryptopunks','superrare', 'unknown']
```

Comment:
Do we know what 'unknown' tokens are / where they come from ?

Reply:
![Screen Shot 2022-07-01 at 10 19 10 AM](https://user-images.githubusercontent.com/9472574/176912414-ea05ab75-d95f-4d6c-834a-4fe14eaa877e.png)
dencentaland wearables?

Reply:
wow ok, i guess that is unknown